index.html

```diff
@@ -83,11 +83,25 @@ <h3>IMSC1.0.1 Compatibility</h3>
     <section>
       <h3>TTML2 Subset</h3>
 
-      <p>Notwithstanding deprecated syntax, as specified below, a valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]]
-      document.</p>
+      <p>A valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]] document.</p>
 
-      <p>[[!ttml-imsc1.0.1]] syntax that is not supported in [[!TTML2]] SHALL be mapped to an equivalent [[!TTML2]] syntax and
-      deprecated in [[!ttml-imsc1.1]].</p>
+      <p>Any [[!ttml-imsc1.0.1]] extension that (i) is not one of the following and (ii) has an equivalent feature in [[!TTML2]]
+      SHALL be deprecated in [[!ttml-imsc1.1]] and the corresponding [[!TTML2]] feature permitted:</p>
+
+      <ul>
+        <li><code>#activeArea</code></li>
+
+        <li><code>#fillLineGap</code></li>
+
+        <li><code>#forcedDisplay</code></li>
+
+        <li><code>#linePadding</code></li>
+
+        <li><code>#multiRowAlign</code></li>
+      </ul>
+
+      <p>When [[!ttml-imsc1.0.1]] syntax is deprecated and mapped to equivalent [[!TTML2]] syntax, both syntaxes SHALL NOT be
+      simultaneously present in the same document.</p>
```

Comment:
We should define "deprecated" to mean "is permitted within a document instance but is expected to be prohibited in a future version".